src/megameklab/com/util/StringUtils.java

```diff
@@ -252,7 +250,7 @@ public static String getAeroEquipmentInfo(Mounted mount) {
         if (mount.getType() instanceof WeaponType) {
             WeaponType weapon = (WeaponType) mount.getType();
             if (weapon instanceof InfantryWeapon) {
-                info = Integer.toString(weapon.getDamage());
+                info = "";
```

Comment:
This is a copy-paste artifact from the ground unit method that adds this info to the damage field. For aerospace units the damage is part of the AV, and this is being appended to the weapon name.